cocotb/_build_libs.py

```diff
@@ -137,14 +137,17 @@ def _gen_import_libs(self, def_dir):
                 )
 
 
-def _extra_link_args(lib_name):
+def _extra_link_args(lib_name=None, rpath=None):
     """
-    Add linker `install_name` argument on osx to load dependencies from the directory
-    where vpi/vhpi/fli library is located
+    Add linker argument to load dependencies from the directory where vpi/vhpi/fli library is located
+    On osx use`install_name`
+    On Linux use`rpath`
     """
 
-    if sys.platform == "darwin":
+    if sys.platform == "darwin" and lib_name is not None:
         return ["-Wl,-install_name,@rpath/%s.so" % lib_name]
+    elif sys.platform == "linux" and rpath is not None:
+        return ["-Wl,-rpath,%s" % rpath]
```

Comment:
Is it safe to do this on darwin too, as
```suggestion
    args = []
    if sys.platform == "darwin" and lib_name is not None:
        args += ["-Wl,-install_name,@rpath/%s.so" % lib_name]
    if rpath is not None:
        args += ["-Wl,-rpath,%s" % rpath]
    return args
```
not that it really